app/data/hooks/rapidpro/IWebhookRepository.ts

```diff
@@ -0,0 +1,8 @@
+import { IVisitor } from '@rocket.chat/apps-engine/definition/livechat';
+import { IUser } from '@rocket.chat/apps-engine/definition/users';
+
+export default interface IWebhookRepository {
```

Comment:
Shouldn't this be renamed from `IWebhookRepository` to `IRapidProWebhookRepository` due to the name of imports? Because `IWebhookRepository` is a very generic name for a webhook repository class.